### DISCUSSION TITLE
Possible bug in DE impl

### DISCUSSION THREAD

**Discussion by PabloAJomer** [mdomke/schwifty#207] [General]
Hi we have discovered a possible bug in DE impl of bban checking. We have another valid account that was incorrectly deemed invalid. I will try and find out why, I can not share the account since it's deemed sensitive information. I will return with more info shortly. Before that do you know if there are any know shortcomings in the DE impl of BBAN validation?

**Comment by mdomke** [comment]
The situation for validating German BBANs is not so easy, because there are so many checksum algorithms for account codes out there. Every bank institute somehow uses their own variant. I used the [official documentation](https://www.bundesbank.de/resource/blob/603320/16a80c739bbbae592ca575905975c2d0/mL/pruefzifferberechnungsmethoden-data.pdf) from the Bundesbank, when I was implementing the algorithms. Maybe you can still tell me which bank the problematic BBAN relates to?
